package.json

```diff
@@ -112,12 +112,12 @@
     "dompurify": "^2.3.7",
     "immutable": "^3.7.3",
     "jsonp": "^0.2.1",
-    "node-fetch": "^2.6.7",
+    "node-fetch": "^2.6.1",
```

Comment:
How come you're downgrading this and do you need this dependency at all?

Reply:
Oh, I am looking into this. I didn't do this manually, I can only assume running `yarn upgrade` did it (see https://github.com/auth0/lock/pull/2209/commits/15ed38157dc7b4701bfa224976f8c80e5891aee8) but not sure why.

Reply:
I've removed this for now as I don't believe it's used anywhere.